### PR TITLE
[onert] Use mmap and munmap in same place

### PR DESCRIPTION
**Pull request by hseok-oh**
Call mmap in MMapedData constructor, call munmap in MMapedData desctructor

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Comment by hseok-oh** [comment]
For
- coverity warning: https://github.com/Samsung/ONE/issues/1635#issuecomment-658032497
- Handle mmap and munmap in `MMapedData` struct

File: runtime/onert/frontend/base_loader/include/base_loader.h [hunk 1]
```diff
@@ -361,7 +364,18 @@ ir::OperandIndex BaseLoader<LoaderDomain, SpecificLoader>::loadOperand(const Ten
...
+    size_t data_size = data->size();
+    ptrdiff_t unaligned_offset_start = data->data() - _base;
+    ptrdiff_t offset_end = unaligned_offset_start + data_size;
+
+    // Calculated aligned offset from base address of mapped region
+    // munmap accepts memory address which is a multiple of the pagesize
+    ptrdiff_t aligned_offset_start = (unaligned_offset_start / _pagesize) * _pagesize;
+    size_t mmap_size = offset_end - aligned_offset_start;
+
+    auto ptr = std::make_unique<ir::MMapedData>(_fd, aligned_offset_start, mmap_size,
```
Comment (YongseopKim) [inline runtime/onert/frontend/base_loader/include/base_loader.h:377]: Is this eventually mmaped twice? Is it your intention?

Comment (hseok-oh) [reply runtime/onert/frontend/base_loader/include/base_loader.h:377]: Yes. mmaped twice (full model file loading, constant operand data), and unmap 
- full file loading: munmap after model file loading (model graph generation) finish
- constant operand data: munmap on `MMapedData` destructor

**Comment by wateret** [comment]
FYI for mmap/munmap mismatch, I classified it as intentional before.

**Pull request merged**